test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -1786,14 +1790,14 @@ private void checkGetRootContent(String whereClause, int maxHtmlLinks,
       assertTrue(query, query.contains(" WHERE " + whereClause));
     }
 
-    assertEquals("text/html; charset=UTF-8", response.contentType);
-    String content = response.content.toString(UTF_8.name());
+    assertEquals("text/html; charset=UTF-8", response.getContentType());
+    String content = response.getOutputStream().toString();
```

Comment:
You either need to supply the ByteArrayOutputStream to the constructor, which is what we usually do, or you need to cast the OutputStream to a ByteArrayOutputStream, which is kinda gross, so that you can still use toString(String). Switching to the platform default character encoding is not OK.